packages/react/src/provider/AnonAadhaarProvider.tsx

```diff
@@ -16,24 +16,45 @@ import React, { Dispatch, SetStateAction } from 'react'
 import { proveAndSerialize } from '../prove'
 import { SerializedPCD } from '@pcd/pcd-types'
 
+// Props for the AnonAadhaarProvider
+export type AnonAadhaarProviderProps = {
+  /**
+   * `children`: The ReactNode elements that form the child components of your application.
+   * This is a standard prop for components that wrap around other components to provide context or styling.
+   */
+  children: ReactNode
+
+  /**
+   * `_useTestAadhaar`: Flag to determine the usage of test Aadhaar data.
+   * Set this to `true` if you want to use test Aadhaar data instead of real data for development or testing purposes.
+   * Defaults to `true` if not explicitly set.
+   */
+  _useTestAadhaar?: boolean
+
+  /**
+   * `_fetchArtifactsFromServer`: Flag to control the source of zk-SNARK artifacts.
+   * When set to `true`, zk-SNARK artifacts are fetched from a specified AWS server, facilitating production.
+   * When set to `false`, artifacts are fetched from a local server, typically used for development and local testing.
+   * Defaults to `true` if not explicitly set.
+   */
+  _fetchArtifactsFromServer?: boolean
```

Comment:
I think we can let users specify the URL of the artifacts (by default it can be the AWS ones)
Right not we are assuming a specific localhost url+port when its not server.

But maybe in the scope for another PR

Reply:
I think that's a good point.
I created an issue for it #158